janis_assistant/management/workflowmanager.py

```diff
@@ -1431,6 +1431,13 @@ def copy_outputs_if_required(self):
                     f"Couldn't find expected output with tag {out.id_}, found outputs ({', '.join(eoutkeys)}"
                 )
                 continue
+
+            # not None but just some empty values like an empty list or empty string
+            if not eout:
+                return Logger.critical(
+                    f"Engine '{self.engine.id()}' didn't return {out.id_} outputs for Janis to copy, skipping"
+                )
+
```

Comment:
so similar to what we already did a step earlier https://github.com/PMCC-BioinformaticsCore/janis-assistant/blob/09ad5466a1dd398ceebfdf17bd237fd6e5ee6009/janis_assistant/management/workflowmanager.py#L1419

But here, we look into each item in the outputs dictionary